test/verify/check-machines

```diff
@@ -4597,12 +4598,15 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
             def create(self):
                 b.click(".pf-c-modal-box__footer button:contains(Create)")
 
-                if (not self.xfail):
+                if not self.xfail:
                     # Creation of volumes might take longer for some pool types
                     if self.pool_type  in ["disk", "netfs"]:
                         b.wait_present(".pf-c-modal-box__footer button.pf-m-in-progress")
                         b.wait_present(".pf-c-modal-box__footer button:contains(Create):disabled")
                     b.wait_not_present("#create-volume-dialog-modal")
+                else:
+                    if "size" == self.xfail_object:
```

Comment:
this is never exercised due to you calling it `size-group` on the other place :) 
Also `self.xfail_object == "size"` feels much more natural.

Reply:
> I see this message:
> 
> ```
> Storage volume size must not exceed the storage pool's capacity (68.40148162841797)
> ```
> 
> Firstly the number is too precise :) I think that up to 2 decimal points is more then enough. Also unit would not hurt. And lastly it is the maximum capacity, not the free space. I was trying this on pool that shows usage `26.56 / 68.4 GiB` so this number should be `41.84 GB`

Oh doh - I had seen these issues and I forgot to update, probably for destructed, thanks for catching that though.